libs/react-numerics/README.md

```diff
@@ -5,6 +5,15 @@ formatted numbers such as currency or telephone numbers.
 
 A [Bitovi React team](https://www.bitovi.com/frontend-javascript-consulting/react-consulting) project.
 
+## Need help or have questions?
```

Comment:
If the docs aren't linked to by the package.json `homepage` then I think the first line in this help section should be a link to the documentation.